eskip/doc.go

```diff
@@ -186,38 +186,43 @@ predicates start with upper case, while filters start with lower case.
 
 Backend
 
-There are three types of backends: a network endpoint address, a shunt or
-a loopback.
+There are four backend types: HTTP endpoint, shunt, loopback and dynamic.
 
-A network endpoint address example:
+HTTP endpoint:
```

Comment:
I would not change this, because it highlights more the "network", that maps in the code to `NetworkBackend` for example and this document is meant for library users that want to understand the code.